### PR TITLE
Feat/pages/user details

### PR DESCRIPTION
**Pull request by khushilms-cs19**

### CODE REVIEW

File: src/components/EngagementHorizontalCard/__tests__/index.test.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import EngagementHorizontalCard from '..';
+import { render } from '@testing-library/react';
+// import { GET_USER_DATA_BY_ID_URL } from '../../../constants/apiEndpoints';
+// import makeRequest from '../../../utils/makeRequest/';
+// import { useParams } from 'react-router-dom';
+// import { default as mockUserData } from '../../../mocks/PeopleDetailsPage';
+
+// jest.mock('../../../utils/makeRequest');
```
Comment (Chetan-MckinseyBEBR) [inline src/components/EngagementHorizontalCard/__tests__/index.test.jsx:9]: Remove comments if not needed

File: src/components/PeopleHorizontalCard/PeopleHorizontalCard.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
+.user-horizontal-card-container{
+  width: 100%;
+  display: flex;
+  /* height: 120px; */
```
Comment (Chetan-MckinseyBEBR) [inline src/components/PeopleHorizontalCard/PeopleHorizontalCard.css:4]: Remove commented code

**Review by Chetan-MckinseyBEBR — COMMENTED** [review]
Added few comments

File: src/components/EngagementHorizontalCard/__tests__/index.test.jsx (hunk 1 above)
Comment (khushilms-cs19) [reply src/components/EngagementHorizontalCard/__tests__/index.test.jsx]: I have removed the comments and refactored the code.

**Review by harshagarwal08 — APPROVED** [review]

**Pull request merged**